### DISCUSSION TITLE
Relationship filter self?

### DISCUSSION THREAD

**Discussion by jaylik** [nautobot/nautobot#2038] [Q&A]
Is it possible somehow to filter objects related to self?

We would like to show only same device interfaces for device inventory items and vice versa (to be able to bind SFP module as inventory item to interface).

**Comment by bryanculver** [comment]
At the moment this isn't available, but an open feature request exists for it: https://github.com/nautobot/nautobot/issues/1593

**Reply by jaylik** [reply]
@bryanculver - Thanks for the info. Is there any available solution to document SFP types for interfaces? Just asking because for different optics types need to be documented somewhere and preferably not in plain text in some description field.

Netbox allows to bind inventory items directly to interfaces.

**Reply by bryanculver** [reply]
The best option today would be to leverage Custom Fields (of type Selection) and assigning them to the interface but we have discussed and will open a feature request on broadening the support of inventory items on more parent object (in this case Interfaces).
